rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -47,8 +49,15 @@ public class JsonRpcWeb3ServerHandler extends SimpleChannelInboundHandler<ByteBu
     private final JsonNodeFactory jsonNodeFactory = JsonNodeFactory.instance;
     private final JsonRpcBasicServer jsonRpcServer;
 
-    public JsonRpcWeb3ServerHandler(Web3 service, List<ModuleDescription> filteredModules) {
+    public JsonRpcWeb3ServerHandler(JsonRpcBasicServer jsonRpcServer) {
```

Comment:
This is only for testing, so we should mark it as `@VisibleForTesting` and make it package private.

Reply:
Done